.github/workflows/docs.yml

```
@@ -60,6 +60,12 @@ jobs:
       - uses: actions/setup-node@820762786026740c76f36085b0efc47a31fe5020 # v7.0.0
         with:
           node-version: "22"
+      - name: Cache Playwright Chromium
+        if: matrix.python-version == '3.12'
```

actionlint (1.7.12)

[error] property "python-version" is not defined in object type {}

(expression)